app/system/languages/zh_CN/messages.php

```diff
@@ -112,114 +112,114 @@
   'Access system settings' => '进入系统设置',
   'No Pagekit %type%' => '',
   'Registration' => '注册',
-  'Folder' => '',
+  'Folder' => '文件夹',
   'Check your email for the confirmation link.' => '检查确认链接的电子邮件。',
-  'Add %type%' => '',
+  'Add %type%' => '增加 %type%',
   'Your user account has been created. Complete your registration, by clicking the link provided in the mail that has been sent to you.' => '您的用户帐户已创建。为了完成您的注册，邮件已发送到您所提供的 Email。',
-  'Approve Account' => '',
+  'Approve Account' => '批准帐户',
   'Add Menu' => '添加菜单',
-  'Hello!' => '',
-  'Mail successfully sent!' => '邮件成功投递！',
+  'Hello!' => '你好',
```

Comment:
Being Chinese myself, I am aware of the difference of the two words. Instead, I am suggesting to be consistent across the board. 

Reference also:

* Line 39: `您的 Pagekit 数据库已为最新！`
* Line 63: `您的用户帐户已被创建并正在等待网站管理员的批准。`
* Line 68: `激活您在 %site% 的账户`
* Line 76: `该权限具有安全隐患，请给您信任的角色。`
... etc.

Doing a quick search, it suggests that there are currently 10 instances where the translation string uses `你` as opposed to 22 instances where the translation string uses `您`.  

Back when I did vBulletin translations, I've always elected to use the formal `您` because it is more professional and presentable for the end user. However, majority of that software was user facing, and since Pagekit is ultimately 'hidden' from the end user (that is, our visitors will likely never see any portion of Pagekit), it doesn't really matter if we choose formal or not. However, IMHO, we need to be consistent across the board and have the same translation in all instances. I have also elected to use English for these messaging so that the Pagekit team can chime in as to what their thoughts are on this subject matter.